packages/react/src/components/Tile/Tile.js

```diff
@@ -542,15 +542,17 @@ export class ExpandableTile extends Component {
 
     return (
       // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
-      <div
+      <button
         ref={tile => {
           this.tile = tile;
         }}
         style={tileStyle}
         className={classes}
+        aria-expanded={isExpanded}
+        title={isExpanded ? tileExpandedIconText : tileCollapsedIconText}
         {...other}
         onClick={this.handleClick}
-        onKeyPress={this.handleKeyDown}
+        // onKeyPress={this.handleKeyDown}
```

Comment:
Did you mean to remove this line, or keep it? Or is this PR still WIP...?

Reply:
This is still a WIP but since I changed the outer `div` to a button, `handleKeydown` is no longer necessary since handleClick will be triggered for keydown events. Definitely meant to remove it completely though! Not just comment it out, my bad!